src/main/java/Event.java

```diff
@@ -0,0 +1,43 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+    String at;
+    String parsedAt;
+
+    /**
+     * Creates event.
```

Comment:
Lack of empty line between the description and parameters section for Javadoc comments as stated in the coding standard.